### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by D-Groenewegen**
Deserves an update.

### CODE REVIEW

**Comment by D-Groenewegen** [comment]
Belated pull request. Expanded, updated and slighty rearranged sections.

File: README.md [hunk 1]
```diff
@@ -5,26 +5,44 @@ An awesome list for Mirador's projects and plugins.
...
-- [Mirador Ruler](https://github.com/UCLALibrary/mirador-ruler) - A Mirador plugin that allows users to overlay a configurable ruler on any window.
-- [Mirador Jump-to-page](https://github.com/sul-dlss/mirador-jump-to-page) - A Mirador plugin that allows users to select a page from a dropdown.
+- Project Mirador:
+     - [Annotations Plugin (M3)](https://github.com/ProjectMirador/mirador-annotations)
+     - [Annotot Endpoint Plugin (M3)](https://github.com/ProjectMirador/mirador-annotot-endpoint-plugin)
+     - [Download Plugin (M3)](https://github.com/ProjectMirador/mirador-dl-plugin)
+     - [Image Tools (M3)](https://github.com/ProjectMirador/mirador-image-tools)
+     - [Share Plugin (M3)](https://github.com/ProjectMirador/mirador-share-plugin)
+     - [See also the plugin demos](https://github.com/ProjectMirador/mirador-plugin-demos)
+- [BSB Mirador Plugins](https://github.com/dbmdz/mirador-plugins) (M2 and M3) - A repository of Mirador plugins maintained by the Bavarian State Library, including
```
Comment (lutzhelm) [inline README.md:17]: All plugins within this particular repo are M2 plugins. There are a lot of BSB plugins for M3 but they'll have to be added later.

File: README.md [hunk 2]
```diff
@@ -5,26 +5,44 @@ An awesome list for Mirador's projects and plugins.
...
+    - CanvasLink - Lets you get a link to currently shown canvas
+    - ImageCropper - Adds an overlay to the canvas for retrieving the image url for the selected area
+    - PhysicalRuler - Adds a ruler with metric or imperial units
+    - ShareButtons
+    - ViewFromUrl - Lets you update the URL to reflect the current view type, manifest and canvas
+- [Annotation Tooltip (M3)](https://github.com/2SC1815J/mirador-annotation-tooltip-plugin)
+- [LDN Plugin](https://github.com/jeffreycwitt/mirador-ldn-plugin) - An experimental plugin for incorporating supplemental IIIF content into Mirador via Linked Data Notifications.
```
Comment (lutzhelm) [inline README.md]: LDN Plugin is M2.

File: README.md [hunk 3]
```diff
@@ -5,26 +5,44 @@ An awesome list for Mirador's projects and plugins.
...
+- [Jump-to-page (M2)](https://github.com/sul-dlss/mirador-jump-to-page) - A Mirador plugin that allows users to select a page from a dropdown.
+- [Metadata Tab plugin](https://github.com/Daniel-KM/Mirador-plugin-MetadataTab) - developed for Omeka S.
```
Comment (lutzhelm) [inline README.md]: Metadata Tab plugin is also M2.

File: README.md [hunk 4]
```diff
@@ -5,26 +5,44 @@ An awesome list for Mirador's projects and plugins.
...
 - [IIIF Toolkit with Mirador](https://github.com/utlib/IiifItems) - Embeds Mirador into Omeka Classic with a built-in annotator, a manifest generator and importer, Simple Pages shortcodes and Exhibit Builder blocks for a rich IIIF-compliant presentation experience.
+- [Mirador extension for MediaWiki](https://github.com/a-g-van-hamel-foundation/Mirador/) - offers a parser functiion to make a build of Mirador 3, including four plugins, available for use in MediaWiki. Developed by CODECS.
 
 ## Packages
 - [mirador_rails](https://github.com/sul-dlss/mirador_rails) - A Ruby gem useful for including Mirador into Ruby on Rails applications
 
 ## Tutorials
 
-- [Tutorial Mirador Viewer](http://heron-net.be/libisplus/themes/LIBIS_PLUS/images/Docs/Tutorial_Mirador.pdf) - PDF explaining how to use many Mirador features.
+- [Tutorial Mirador Viewer](http://heron-net.be/libisplus/themes/LIBIS_PLUS/images/Docs/Tutorial_Mirador.pdf) - PDF explaining how to use many Mirador features. Accessible only after login authentication.
```
Comment (lutzhelm) [inline README.md:45]: Do we know if this actually can be found behind the login? Maybe @jronallo who added the entry originally knows more.

File: README.md [hunk 5]
```diff
@@ -5,26 +5,44 @@ An awesome list for Mirador's projects and plugins.
...
+    - [Disable Zoom](https://github.com/UCLALibrary/mirador-disable-zoom) - A Mirador plugin that allows users to disable (and re-enable) zoom controls per window.
+    - [Sync Windows](https://github.com/UCLALibrary/mirador-sync-windows) - A Mirador plugin that allows users to mirror actions performed in one window in one or more other windows.
+    - [Share Workspace](https://github.com/UCLALibrary/mirador-share-workspace) - A Mirador plugin that allows users to share workspaces across workstations.
+    - [Ruler](https://github.com/UCLALibrary/mirador-ruler) - A Mirador plugin that allows users to overlay a configurable ruler on any window.
 
 ## Mirador-Based Tools
 
 - [IIIF Toolkit with Mirador](https://github.com/utlib/IiifItems) - Embeds Mirador into Omeka Classic with a built-in annotator, a manifest generator and importer, Simple Pages shortcodes and Exhibit Builder blocks for a rich IIIF-compliant presentation experience.
+- [Mirador extension for MediaWiki](https://github.com/a-g-van-hamel-foundation/Mirador/) - offers a parser functiion to make a build of Mirador 3, including four plugins, available for use in MediaWiki. Developed by CODECS.
```
Comment (lutzhelm) [inline README.md]: Typo: one "i" to many "functiion"

**Review by lutzhelm — CHANGES_REQUESTED** [review]
Thanks for getting at this!
If you could just add the additional M2 indicators and remove the typo I would merge the PR.

I think we'll handle the PDF tutorial later. #12 is about that.

File: README.md (hunk 1 above)
Comment (D-Groenewegen) [reply README.md:17]: Now I remember those M3 plugins are to be found in repos of their own. One for the to-do list.

File: README.md (hunk 4 above)
Comment (D-Groenewegen) [reply README.md:45]: Maybe remove the link as the PDF would be unavailable for most people? By now, there are many tutorials, especially video tutorials, out there that could be included instead.

Comment (lutzhelm) [reply README.md:45]: Yes, that would probably be better for now.

File: README.md (hunk 1 above)
Comment (jbaiter) [reply README.md:17]: If it helps, here is a list of the plugins we currently have:
- https://github.com/dbmdz/mirador-textoverlay
- https://github.com/dbmdz/mirador-keyboardshortcuts
- https://github.com/dbmdz/mirador-imagecropper
- https://github.com/dbmdz/mirador-downloaddialog
- https://github.com/dbmdz/mirador-canvaslink
- https://github.com/dbmdz/mirador-canvasnavigation

**Review by lutzhelm — APPROVED** [review]

**Pull request merged**